src/tcgwars/logic/impl/gen3/DeltaSpecies.groovy

```diff
@@ -3040,7 +3075,7 @@ public enum DeltaSpecies implements LogicCardInfo {
         pokePower "Evolutionary Swirl", {
           text "Once during your turn, when you play Vaporeon ex from your hand to evolve 1 of your Pokémon, you may have your opponent shuffle his or her hand into his or her deck. Then, your opponent draws up to 4 cards."
           onActivate {
```

Comment:
"Your opponent draws up to 4 cards." Can the opponent choose the number of cards they draw then?

Reply:
Good catch, tend to miss those texts since modern cards don't really offer that choice :P